src/status_im/chat/models/message_list.cljs

```diff
@@ -16,16 +16,20 @@
                                message-type
                                from
                                outgoing
-                               whisper-timestamp]}]
+                               whisper-timestamp
+                               deleted-for-me?]}]
   (-> {:whisper-timestamp whisper-timestamp
-       :from from
-       :one-to-one? (= constants/message-type-one-to-one message-type)
-       :system-message? (= constants/message-type-private-group-system-message
-                           message-type)
-       :clock-value clock-value
-       :type :message
-       :message-id message-id
-       :outgoing (boolean outgoing)}
+       :from              from
+       :one-to-one?       (= constants/message-type-one-to-one message-type)
+       :system-message?   (boolean
+                           (or
+                            (= constants/message-type-private-group-system-message
+                               message-type)
+                            deleted-for-me?))
```

Comment:
Count deleted-for-me message as system message in status-mobile only.
It's still user's message in status-go, since we need to since this between paired devices.